internal/daemon/metric/initialize_metrics.go

```diff
@@ -33,7 +33,18 @@ var (
 
 /* The following methods are used to initialize Prometheus histogram vectors for gRPC connections. */
 
-func rangeProtoFiles(m map[string][]string, fd protoreflect.FileDescriptor) bool {
+func contains(r []string, n string) bool {
```

Comment:
You could also make use of the go-secure-stdlibs we have created for this https://github.com/hashicorp/go-secure-stdlib/blob/main/strutil/strutil.go#L26

Reply:
ah thanks, I was wondering if there was a better way to do this